<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VenCalcu</class>
 <widget class="QWidget" name="VenCalcu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Calculadora Básica</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Mi_Primer_GUI_PyQt5/Recursos/MCServer-icon-64x64.png</normaloff>../Mi_Primer_GUI_PyQt5/Recursos/MCServer-icon-64x64.png</iconset>
  </property>
  <property name="statusTip">
   <string notr="true">Listo.</string>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Peru"/>
  </property>
  <widget class="QPushButton" name="bNum1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 1</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">1</string>
   </property>
   <property name="shortcut">
    <string notr="true">1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 2</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">2</string>
   </property>
   <property name="shortcut">
    <string notr="true">2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 4</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">4</string>
   </property>
   <property name="shortcut">
    <string notr="true">4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 5</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">5</string>
   </property>
   <property name="shortcut">
    <string notr="true">5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 3</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">3</string>
   </property>
   <property name="shortcut">
    <string notr="true">3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 6</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">6</string>
   </property>
   <property name="shortcut">
    <string notr="true">6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum9">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 9</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">9</string>
   </property>
   <property name="shortcut">
    <string notr="true">9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 7</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">7</string>
   </property>
   <property name="shortcut">
    <string notr="true">7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum8">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 8</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">8</string>
   </property>
   <property name="shortcut">
    <string notr="true">8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bPorcentaje">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Porcentaje</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">%</string>
   </property>
   <property name="shortcut">
    <string notr="true">%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bNum0">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 0</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">0</string>
   </property>
   <property name="shortcut">
    <string notr="true">0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bPunto">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Punto Decimal</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">.</string>
   </property>
   <property name="shortcut">
    <string notr="true">.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boCalcular">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>280</y>
     <width>111</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Calcular Operación</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">=</string>
   </property>
   <property name="shortcut">
    <string notr="true">Enter</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boRestar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Signo Menos</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">-</string>
   </property>
   <property name="shortcut">
    <string notr="true">-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boBorrar">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 8</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">&lt;-</string>
   </property>
   <property name="shortcut">
    <string notr="true">Backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bsParentAbrir">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Abrir Paréntesis</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">(</string>
   </property>
   <property name="shortcut">
    <string notr="true">(</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boSumar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Signo Suma</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">+</string>
   </property>
   <property name="shortcut">
    <string notr="true">+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boRaiz">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Signo raíz cuadrada</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">√</string>
   </property>
   <property name="shortcut">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="boDividir">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>280</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Signo Fracción</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">/</string>
   </property>
   <property name="shortcut">
    <string notr="true">/</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bsParentCerrar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Cerrar Paréntesis</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">)</string>
   </property>
   <property name="shortcut">
    <string notr="true">)</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boLimpiar">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Limpiar Pantalla</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">C</string>
   </property>
   <property name="shortcut">
    <string notr="true">Del, Backspace</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boMultiplicar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Signo Por</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">*</string>
   </property>
   <property name="shortcut">
    <string notr="true">*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="boPotencia">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Número 2</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true"/>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="locale">
    <locale language="Spanish" country="Peru"/>
   </property>
   <property name="text">
    <string notr="true">x²</string>
   </property>
   <property name="shortcut">
    <string notr="true">Ctrl+Alt+^, </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pantallaCalcu">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>431</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Cambria Math</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string notr="true">Pantalla</string>
   </property>
   <property name="statusTip">
    <string notr="true"/>
   </property>
   <property name="whatsThis">
    <string notr="true">Pantalla de la calculadora</string>
   </property>
   <property name="accessibleName">
    <string notr="true"/>
   </property>
   <property name="accessibleDescription">
    <string notr="true"/>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="inputMask">
    <string notr="true"/>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="alignment">
    <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
   <property name="placeholderText">
    <string notr="true">Esperando operación...</string>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>bNum0</tabstop>
  <tabstop>bPunto</tabstop>
  <tabstop>bPorcentaje</tabstop>
  <tabstop>bNum1</tabstop>
  <tabstop>bNum2</tabstop>
  <tabstop>bNum3</tabstop>
  <tabstop>bNum4</tabstop>
  <tabstop>bNum5</tabstop>
  <tabstop>bNum6</tabstop>
  <tabstop>bNum7</tabstop>
  <tabstop>bNum8</tabstop>
  <tabstop>bNum9</tabstop>
  <tabstop>boSumar</tabstop>
  <tabstop>boBorrar</tabstop>
  <tabstop>boLimpiar</tabstop>
  <tabstop>boRestar</tabstop>
  <tabstop>bsParentAbrir</tabstop>
  <tabstop>bsParentCerrar</tabstop>
  <tabstop>boMultiplicar</tabstop>
  <tabstop>boPotencia</tabstop>
  <tabstop>boRaiz</tabstop>
  <tabstop>boDividir</tabstop>
  <tabstop>boCalcular</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>boLimpiar</sender>
   <signal>pressed()</signal>
   <receiver>pantallaCalcu</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>447</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
